apps/web/pages/_app.tsx

```diff
@@ -44,6 +44,10 @@ function MyApp(props: AppProps) {
       <Head>
         <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />
       </Head>
+      <Script
+        id="calcom-version"
+        dangerouslySetInnerHTML={{ __html: `window.CalComVersion = '${CalComVersion}'` }}
+      />
```

Comment:
@hariombalhara where do we use this?

Reply:
Not anywhere yet. But it might be useful for other tools on-page to know what version cal.com is running on.